examples/taskruns/s3-tgz-input.yaml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
These examples will get run as part of the "YAML tests" during the CI integration suite. Since we don't have buckets set up on AWS (I don't think) I don't expect these examples will pass testing.

Reply:
They might work if the objects are publicly accessible, that's how the GCS YAML tests work.

Reply:
One thing I thought about - since this uses gocloud, we could just use an inmem cluster URL instead of S3 url and it all might just work.

Reply:
ah like minio running in a sidecar or something?  :+1:

Reply:
I think not hitting something in the internet is always a better option in a CI job, so I like the idea of a sidecar.

Otherwise afaik GCS buckets can also be accessed via an S3 interface, so we could use the same bucket used by the GCS test.